app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -66,6 +67,19 @@ class PipelineSampleReads extends React.Component {
     this.toggleHighlightTaxon = this.toggleHighlightTaxon.bind(this);
   }
 
+  generateGsnapFilterStatus(jobStats) {
+    if (this.host_genome && this.host_genome.name === "Human") {
+      // only relevant for Human  as of 5/21/2018
+      for (let stat of jobStats) {
+        if (stat["task"] === "run_gsnap_filter") {
```

Comment:
is the reason for this like "if the gsnap filter is running, then there's no status yet?"